pkg/subctl/operator/submarinerop/scc/ensure.go

```diff
@@ -23,5 +23,11 @@ import (
 )
 
 func Ensure(restConfig *rest.Config, namespace string) (bool, error) {
-	return scc.UpateSCC(restConfig, namespace, serviceaccount.OperatorServiceAccount)
+	operatorUserAdded, err := scc.UpateSCC(restConfig, namespace, serviceaccount.OperatorServiceAccount)
+	if err != nil {
+		return false, err
+	}
+
+	npSyncerUserAdded, err := scc.UpateSCC(restConfig, namespace, serviceaccount.NPSyncerServiceAccount)
```

Comment:
This one doesn't need to be on the SCC, it hopefully won't run as a privileged pod. (as far as I have identified)

Reply:
ok - I wasn't sure what this stuff was for 😄

Reply:
In openshift there are security contexts for service accounts, and for being allowed to escalate for use of hostnetwork pod, or privileged ports, you need to be on the privileged security context. ':)